frontend/lib/src/components/elements/ArrowVegaLiteChart/ArrowVegaLiteChart.tsx

```diff
@@ -590,17 +592,22 @@ export class ArrowVegaLiteChart extends PureComponent<
       throw this.state.error
     }
 
+    // Create the container Vega draws inside.
```

Comment:
super-nit: Create the container inside which Vega draws its content

Reply:
Changed 👍